src/main/java/teammates/common/datatransfer/SearchResultBundle.java

```diff
@@ -1,45 +1,14 @@
 package teammates.common.datatransfer;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-
-import teammates.common.util.Const;
-
-import com.google.appengine.api.search.Results;
-import com.google.appengine.api.search.ScoredDocument;
-
 /**
- * The search result bundle object.
+ * The basic search result bundle object.
  */
-public abstract class SearchResultBundle {
-    protected String extractContentFromQuotedString(String quotedString) {
-        if (quotedString.matches("^\".*\"$")) {
-            return quotedString.substring(1, quotedString.length() - 1);
-        }
-        return quotedString;
-    }
+public class SearchResultBundle {
```

Comment:
Not possible; interfaces cannot have any variable.
Abstract class is not possible either because there is no abstract method.